share/spice/amazon/amazon.js

```diff
@@ -16,8 +16,9 @@
         };
 
         if (api_result.results.length > 1) {
+            var isBrandExp = DDG.page.ads.adxExperiment === 'prod_brand_v1';
```

Comment:
We can use env here over DDG.

Also I'm pretty sure that env.page.ads will never cause a null reference error here, but it's worth double, double checking!

Reply:
> We can use env here over DDG.

@bbraithwaite I don't think so? AFAIK `env === window` here. So it'd need to be `env.DDG.ads`

>but it's worth double, double checking!

I'll add an extra check 👍

Reply:
@moollaza ah you're right re: `DDG` vs `env`, I'm so used to the internal code creating the IIFE with (DDG). 👍